search-service/src/main/kotlin/com/egm/stellio/search/csr/model/Operation.kt

```diff
@@ -1,8 +1,15 @@
 package com.egm.stellio.search.csr.model
 
 import com.fasterxml.jackson.annotation.JsonProperty
+import org.springframework.http.HttpMethod
+import java.net.URI
 
-enum class Operation(val key: String, private val matchingOperationGroups: Set<Operation> = emptySet()) {
+enum class Operation(
+    val key: String,
+    private val matchingOperationGroups: Set<Operation> = emptySet(),
+    val method: HttpMethod? = null,
+    private val path: NGSILDPath? = null
```

Comment:
some thoughts:
- a method and a path for an operation is actually not specific to dist ops and could be defined in a shared place (for instance, we have a bunch of hardcoded paths that could benefit from this)
- still not fan of the `!!` here and there but I can live with them for now (called in controlled places)